tasks/configure.yml

```diff
@@ -57,7 +57,7 @@
     dest: "{{ prometheus_config_dir }}/file_sd/"
     force: true
     owner: root
-    group: prometheus
+    group: "{% if prometheus_docker_install %} 65534 {% else %} prometheus {% endif %}"
```

Comment:
Such operations should be set in preflight.yml and here it should be only a reference to some registered fact. Even better if those were set in some file in `vars/` directory.

Reply:
got it ! :)